haas/ext/obm/haas_obm.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2015 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+""" An obm driver for recursive HaaS.
+It invokes Base HaaS to execute various OBM actions on nodes 
+"""
+
+from sqlalchemy import Column, String, Integer, ForeignKey
+import schema
+import requests
+import logging
+
+
+from haas.model import Obm
+from haas.errors import OBMError
+from haas.dev_support import no_dry_run
+from haas.config import cfg
+
+
+class Haas_obm(Obm):
```

Comment:
Currently the 'config parser' converts all class names to lowercase. I have made another pull request #497 that enables camel case. Once this request is accepted I will implement the suggestion made here.